bingo.js

```diff
@@ -347,8 +347,9 @@ function forEachSquare(f)
 
 function generateNewSheet()
 {
-	$(".seed_for_copying").val(SEED);
-	$(".seed_for_copying").attr("size", SEED.length);
+	let seedInput = $(".seed-input");
+	seedInput.val(SEED);
+	seedInput.attr("size", Math.max(SEED.length, 1));
```

Comment:
```suggestion
	seedInput.attr("size", Math.max(SEED.length, 5));
```

I recommend making the minimum size bigger so that the field and the whole row of controls is less jumpy when hitting backspace in the field.

Reply:
compare one:

https://user-images.githubusercontent.com/624072/198126278-a1706258-60d9-4a59-bc08-093267265a2c.mp4

vs five:

https://user-images.githubusercontent.com/624072/198126301-4a119791-ee4e-40ec-b5ae-447a6b11581f.mp4

Reply:
This makes more sense, thank you.